recipes/moabs/meta.yaml

```diff
@@ -5,11 +5,11 @@ package:
   version: {{ version }}
 
 build:
-  number: 0
+  number: 1
```

Comment:
if you upgrade a version you need to reset the build number to 0

Reply:
Sure. Thank you for the changes made in the PR.